tests/e2e-legacy/since-release.mjs

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env node
+
+import { join } from "path";
+import { execSync } from "child_process";
+import { getDirName } from "./getDirName.mjs";
+import { getAllTags } from "./getAllTags.mjs";
+import { getPackageTags } from "./getPackageTags.mjs";
+import { runTestForTags } from "./runTestForTags.mjs";
+
+const __dirname = getDirName();
+const ROOT_BIN = join(__dirname, "..", "..", "node_modules", ".bin");
+
+console.info(`Looking for changed packages...`);
+let changedPackages = [];
+try {
+  const execOptions = { ...process, cwd: __dirname, encoding: "utf-8" };
+  changedPackages = execSync(`${join(ROOT_BIN, "lerna")} changed`, execOptions).split("\n");
+} catch (e) {
+  // Swallow error because Lerna throws if no package changes.
```

Comment:
I suggest matching against expected error string, throwing for other errors

Reply:
This is from existing code https://github.com/aws/aws-sdk-js-v3/blob/021f2181d3fa7d3a2760c97b66ab76f68c70e2f6/tests/e2e-legacy/index.js#L21-L25

There's no specific error thrown by lerna
```console
$ git log --oneline | head -n3
021f2181d3f Publish v3.663.0
9a58dedd740 feat(clients): update client endpoints as of 2024-10-02
5dd5cd26370 feat(client-b2bi): Added and updated APIs to support outbound EDI transformations

$ yarn lerna changed
yarn run v1.22.22
$ /local/home/trivikr/workspace/aws-sdk-js-v3/node_modules/.bin/lerna changed
lerna notice cli v5.5.2
lerna notice Current HEAD is already released, skipping change detection.
lerna info No changed packages found
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```